tests/cover/test_one_of.py

```diff
@@ -17,14 +17,11 @@
 
 from __future__ import division, print_function, absolute_import
 
-import pytest
-
 import hypothesis.strategies as st
-from hypothesis.errors import NoExamples
+from tests.common.debug import assert_no_examples
 
 
 def test_one_of_empty():
     e = st.one_of()
     assert e.is_empty
-    with pytest.raises(NoExamples):
-        e.example()
```

Comment:
Apparently this was the only test covering [this line](https://github.com/HypothesisWorks/hypothesis-python/blob/master/src/hypothesis/searchstrategy/strategies.py#L323)

Reply:
It was, but the change shouldn't have affected whether or not it was covered! I'll dig into why.

Reply:
Oh, I see, because we wrap things inside tuples when using them in given, and that causes this to fire: https://github.com/HypothesisWorks/hypothesis-python/blob/master/src/hypothesis/strategies.py#L394